common/src/BasePlugin.php

```diff
@@ -36,6 +37,13 @@ public function init() {
         add_action('plugins_loaded', [$this, 'loadTranslations']);
         add_action('admin_enqueue_scripts', [$this, 'addUpdateNoticeDismissScript']);
         add_action('wp_ajax_' . $this->getUpdateNoticeDismissedAjaxHook(), [$this, 'dismissUpdateNotice']);
+        add_action('before_woocommerce_init', function() {
+            if (class_exists(FeaturesUtil::class)) {
+                FeaturesUtil::declare_compatibility(
+                    'custom_order_tables', $this->getSlug() . '/' . $this->getSlug() . '.php', true
```

Comment:
`getPluginFile` will return the same thing, and also handle the situation where the plugin directory is renamed. This is possible in WordPress, and it should still work in that case.

```suggestion
                    'custom_order_tables', $this->getPluginFile(), true
```